open_spiel/python/tests/policy_test.py

```diff
@@ -50,6 +50,56 @@
 ]
 
 
+class DerivedPolicyTest(absltest.TestCase):
+    def test_derive_from_policy(self):
+        try:
+            # equivalent to:
+            # class DerivedPolicy(pyspiel.Policy):
```

Comment:
This would be much clearer; is there any reason not to use it?

Reply:
I wasn't sure whether class statements may be parsed by Python at an earlier stage than execution, so I just opted for type which I knew to be dynamic. Changed it to the clearer declaration.